container.yaml

```diff
@@ -9,4 +9,3 @@ compose:
 platforms:
   only:
   - x86_64
-  - s390x
```

Comment:
Do not remove, only comment out this line.

Reply:
@drichtarik Yes, it's probably better to keep it, just comment out. Also for the _content_sets.yaml_ file, you might want to comment out the s390x entry (just for the case it would case issues later for the build). Otherwise LGTM

Reply:
No, please do not change content_sets.yaml.